README.md

```diff
@@ -5,94 +5,90 @@
-
-Remaining items to implement are:
-
-- Migrations from non-SecureBoot machine (old generation files) ;
-- Alternative Nix stores paths ;
-- Key rotation support ;
-- Bootspec (abuse) cleanups ;
-- Automatic synchronization policies for changing PKI (rotating keys, re-enrolling them, etc.) ;
-- NixOS specialisations support ;
-- Automatic removal of unused files relative to the `configurationLimit` option ;
-- `os-release` patch so `systemd-boot` shows pretty names with generation number
-
-## lanzaboote
-
-`lanzaboote` is the stub that `lanzatool` uses to form an UKI. It
-loads a Linux kernel and initrd without breaking the Secure Boot chain
-of trust. Instead of rolling our own crypto, `lanzaboote` re-uses the
-signature verification that is built-in to UEFI.
-
-Remaining items to implement are:
-
-- TPM measurements like `systemd-stub` does
-- Better error management
-
-## Relevant Nixpkgs Work
-
-This project depends on upstream nixpkgs work:
-
-- https://github.com/NixOS/nixpkgs/pull/191665
-- https://github.com/DeterminateSystems/bootspec-secureboot/
-- https://github.com/DeterminateSystems/bootspec
-
-You can find everything integrated as PoC
-[here](https://github.com/NixOS/nixpkgs/pull/202497).
+This repository contains tooling for [UEFI Secure
+Boot](https://en.wikipedia.org/wiki/UEFI#Secure_Boot) on
+[NixOS](https://nixos.org/). The goal is to make Secure Boot available
+from [nixpkgs](https://github.com/NixOS/nixpkgs) for any platform that
+supports UEFI.
+
+## ⚡ Quickstart ⚡
+
+If you want to try this out, head over [here](./docs/QUICK_START) for
+instructions.
+
+## 🪛 Get Involved 🪛
+
+There is still a bunch of work to do before this work can be
+upstreamed into [nixpkgs](https://github.com/NixOS/nixpkgs). Please
+coordinate in the [Matrix
+room](https://matrix.to/#/#nixos-secure-boot:ukvly.org) or check the
+[issues](https://github.com/nix-community/lanzaboote/issues), if you
+want to take something up.
+
+## Overview
+
+### Secure Boot
+
+The goal of UEFI Secure Boot is to allow only trusted operating
+systems to boot on a system. This can be used to defend against
+certain classes of attacks that compromise the boot flow of a
+system. For example, an attacker will have difficulty replacing the
+Linux kernel that boots a system when Secure Boot is active.
+
+UEFI Secure Boot works by digitally signing all drivers, bootloaders,
+the Linux kernel and its initrd. This establishes a chain of trust
+where one trusted component only hands off control to the next part of
+the boot flow when the integrity of the chain is cryptographically
+validated.
+
+### Caveats
+
+There are some additional steps that are required to make UEFI Secure
+Boot effective:
+
+- There must be a BIOS password or a similar restriction that prevents
+  unauthorized changes to the Secure Boot policy.
+- The booted system must have some form of integrity protection.
+
+These steps will not be covered here.
```

Comment:
would add the nitpick that it does not defend against lower level threats such as compromised firmware, etc. and those are the responsibility of Intel BootGuard or similar, thus, keeping "your machine" updated as in: running fwupd regularly, is important too.
But that might be too advanced, and we can elaborate on this somewhere else.